auth/user_mgt_test.go

```diff
@@ -700,13 +725,31 @@ func echoServer(resp interface{}, t *testing.T) *mockAuthServer {
 	s.Srv = httptest.NewServer(handler)
 	authClient, err := NewClient(context.Background(), &internal.AuthConfig{})
 	if err != nil {
-		t.Fatal()
+		t.Fatal(err)
 	}
 	authClient.url = s.Srv.URL + "/"
 	s.Client = authClient
 	return &s
 }
 
+func buildVersionTestServer(version string, t *testing.T) *mockAuthServer {
```

Comment:
Instead of this, can we just put something permanently into the `mockAuthServer`? That way we don't have to have a new test case.

Reply:
is this what you meant ?